lib/smart_answer/erb_renderer/question_options_helper.rb

```diff
@@ -3,5 +3,9 @@ module ErbRenderer::QuestionOptionsHelper
     def options(new_options = nil)
       @options = new_options || @options || {}
     end
+
+    def hide_caption(hide_caption = false)
+      @hide_caption = hide_caption || @hide_caption
+    end
```

Comment:
I prefer the setter (`hide_caption`) and getter (`hide_caption?`) pairing you had previously

Reply:
I do to - but I think I'm going to keep as it consistent with pattern of options above.